test/integration/dockerapi_fetcher_test.go

```diff
@@ -0,0 +1,39 @@
+package integration_test
+
+import (
+	"context"
+	"encoding/json"
+	"testing"
+
+	"github.com/docker/docker/client"
+	"github.com/newrelic/nri-docker/src/raw/dockerapi"
+	"github.com/stretchr/testify/require"
+)
+
+func TestDockerAPIHelpers(t *testing.T) {
```

Comment:
We should extend it once the fetcher is fetching something.

Reply:
TODO added here: [a086ccd](https://github.com/newrelic/nri-docker/pull/201/commits/a086ccdf8fa3ce72d63bfc3841851b3af1bb60f4). Thanks! 🙂